test/tests/php-fpm-hello-web/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# Build a client image with cgi-fcgi for testing
+client_image="php-fpm-client-$RANDOM-$RANDOM"
+docker build -q -t "$client_image" - > /dev/null <<EOF
```

Comment:
And now that I spell it out, the "fat" idea seems dumb since it would be some ad-hoc thing for people to make client image `FROM` that is only for official image tests.